src/components/ose/NewServiceAccount.vue

```diff
@@ -3,10 +3,10 @@
         <div class="hero is-light">
             <div class="hero-body">
                 <div class="container">
-                    <h1 class="title"><i class="material-icons">account_circle</i> Service-Account anlegen</h1>
+                    <h1 class="title"><i class="material-icons">account_circle</i>Create  service-account</h1>
                 </div>
                 <h2 class="subtitle">
-                    Hier kannst du für dein Projekt einen Service-Account anlegen</h2>
+                    Here you can create a project for your service-account</h2>
```

Comment:
```suggestion
                    Here you can create a service account for your project</h2>
```
Alles andere ergibt irgendwie keinen Sinn :smirk: